samples/sensor/adxl345/sample.yaml

```diff
@@ -0,0 +1,27 @@
+sample:
+  name: ADXL345 Sensor sample
+tests:
+  sample.sensor.adxl345.i2c:
+    harness: console
+    tags: sensors
+    depends_on: i2c
+    integration_platforms:
+      - nucleo_f030r8
+    platform_allow: nucleo_f030r8
```

Comment:
Please filter based on sensor dt compatible being enabled. This sample is not board specific.

Reply:
Updated. Please verify it.